api/services/Websockets.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function (room, messageType, payload, socketToExclude) {
```

Comment:
let's anticipate more functions ending up in this file, and export a named function instead of having it be the default export